app/javascript/components/ingredients/ingredients-form.vue

```diff
@@ -129,31 +165,47 @@ export default {
         sku: null,
         price: '',
         currency: 'CLP',
-        quantity: '',
-        measure: '',
+        ingredient_measures_attributes: [],
       },
+      showingMeasureModal: false,
+      usedUnits: [],
     };
   },
-
+  methods: {
+    AddMeasure() {
+      const last_item = this.form.ingredient_measures_attributes[this.form.ingredient_measures_attributes.length -1];
+      if(last_item.name && last_item.quantity){
+        this.form.ingredient_measures_attributes.push({ name: undefined, quantity: undefined, id: undefined });
+      }
+    },
+    deleteUnit(unit) {
+      this.form.ingredient_measures_attributes = this.form.ingredient_measures_attributes.filter((originalUnit) => originalUnit !== unit);
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  This line has a length of 138. Maximum allowed is 120             max-len